### ISSUE TITLE
USWDS - Bug: sorted styles applied when aria-sort=none

### ISSUE THREAD

**Issue by szeinedd-nlr** [uswds/uswds#6196]
### Describe the bug

I am using `aria-sort` on a table's `th` element. The sorted style (blue background) is getting applied to the `th` even when `aria-sort` is set to `none`.

### Steps to reproduce the bug

1. Create a table
2. Add `aria-sort="none"` to one of the th elements

```html
      <table class="usa-table usa-table--striped usa-table--borderless">
        <thead>
          <tr>
            <th role="columnheader" scope="col" aria-sort="none">Heading</th>
            <th role="columnheader" scope="col" aria-sort="none">Heading</th>
            <th role="columnheader" scope="col" aria-sort="none">Heading</th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>value</td>
            <td>value</td>
            <td>value</td>
          </tr>
        </tbody>
      </table>
```

### Expected Behavior

adding `aria-sort="none"` should have no effect on the style

### Related code

https://github.com/uswds/uswds/blob/f0684a3812d85d26c3868c9368ec8bf1689a0d52/packages/uswds-core/src/styles/mixins/general/table.scss#L288

### System setup

USWDS 3.9.0

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by mejiaj** [comment]
Assigning to next sprint for confirmation.
